array-benchmark-100.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-echo "AllocateInJavaGetArray,JniGetArray,Jni2DGetArray" > array_benchmark.csv
+echo "AllocateInJavaGetArray,JniGetArray,Jni2DGetArray" > array-benchmark.csv
 for i in `seq 1 100`;
 do
-  echo "ArrayBenchmark run $i/100"
-  target/jni-construction-benchmark-1.0-SNAPSHOT-application/jni-construction-benchmark-1.0-SNAPSHOT/bin/benchmark --benchmark=ArrayBenchmark --csv --iterations=1000000 >> array_benchmark.csv
+  echo -en "\rArrayBenchmark run $i/100"
```

Comment:
Interesting! What is the purpose of using `-en` and an explicit `\r` as opposed to just `echo` which will print a new line?

If I remember correctly, Windows line endings are `\r\n, whilst posix is just `\n`.

Reply:
It will print on the same line (overwrite it) - looks better and you do not lose previous terminal output from sight.
From `echo` manual:

>     Options:
>       -n	do not append a newline
>       -e	enable interpretation of the following backslash escapes

Reply:
Neat! :-)